partcad-cli/src/partcad_cli/click/commands/add/sketch.py

```
@@ -0,0 +1,55 @@
+    help="The part description (also used by LLMs).",
+    required=False,
+    show_envvar=True,
+)
+@click.option(
+    "--ai",
+    "provider",
+    type=click.Choice(
+        [
+            "google",
+            "openai",
+        ]
```

Ruff (0.8.2)

Trailing comma missing

Add trailing comma

(COM812)